### DISCUSSION TITLE
Get reason for failed listen

### DISCUSSION THREAD

**Discussion by Anigif** [uNetworking/uWebSockets.js#545] [Q&A]
I've developed a nodejs with uWebsocket.js locally and it works quite well. When I push it to my server it however fails - when listening on port 80 the listenSocket is simply false. Is there any way I can figure out why it fails? Haven't found anything in the docs or this forum indicating how to. 
Thanks in advance :)

**Comment by e3dio** [comment]
I dont know if you can get reason but port 80 linux needs extra settings to run on, I use `sudo sysctl net.ipv4.ip_unprivileged_port_start=80`

**Reply by Anigif** [reply]
Interesting, might be the cause. Thanks for the tip! I'm not that used to working with NodeJS and I'm doing it via cPanel (in my webhotel) - so I'm not really sure if I can set that property.
I previously used fastify and it worked when listening on port 80, but might be difference since fastify is using the default http package?

**Reply by e3dio** [reply]
check if something else using the port, ssh in to run commands

**Reply by Anigif** [reply]
That might be the cause - it's working with other ports :) I was just wondering since it was working locally, but guess the webhotel is just using port 80 pr. default, which makes sense. Thanks for the help! :)
